src/Stratis.Bitcoin.IntegrationTests/API/ApiSteps.cs

```diff
@@ -581,7 +581,7 @@ private IActionResult BuildTransaction()
                     Password = WalletPassword,
                     WalletName = WalletName,
                     FeeAmount = Money.Satoshis(82275).ToString() // Minimum fee
-                });
+                }).GetAwaiter().GetResult();
```

Comment:
This doesnt need to be here?

Reply:
Yes I think so, Its a test and the WalletController is now Async

Reply:
Done